src/span.js

```diff
@@ -6,6 +6,88 @@ let defaultTracer = require('./default_tracer');
 const kKeyRegExp = new RegExp(/^[a-z0-9][-a-z0-9]*/);
 
 /**
+ * SpanContext represents Span state that must propagate to descendant Spans
+ * and across process boundaries.
+ *
+ * SpanContext is logically divided into two pieces: the user-level "Baggage"
+ * (see setBaggageItem and getBaggageItem) that propagates across Span
+ * boundaries and any Tracer-implementation-specific fields that are needed to
+ * identify or otherwise contextualize the associated Span instance (e.g., a
+ * <trace_id, span_id, sampled> tuple).
+ */
+export default class SpanContext {
```

Comment:
There can only be one `default` export per file. For consistency, this should likely be in its own file.  I'm a little surprised this didn't generate some kind of error...